dev/MenuBar/MenuBarItem.cpp

```diff
@@ -160,10 +160,42 @@ void MenuBarItem::OnMenuBarItemKeyDown( winrt::IInspectable const& sender, winrt
     {
         ShowMenuFlyout();
     }
+    else if (key == winrt::VirtualKey::Right)
+    {
+        if (FlowDirection() == winrt::FlowDirection::RightToLeft)
+        {
+            MoveFocusTo(FlyoutLocation::Left);
+        }
+        else
+        {
+            MoveFocusTo(FlyoutLocation::Right);
+        }
+    }
+    else if (key == winrt::VirtualKey::Left)
+    {
+        if (FlowDirection() == winrt::FlowDirection::RightToLeft)
+        {
+            MoveFocusTo(FlyoutLocation::Right);
+        }
+        else
+        {
+            MoveFocusTo(FlyoutLocation::Left);
+        }
+    }
+
+    args.Handled(TRUE);
```

Comment:
It is best practice to only mark as handled if you actually handle the key press.

Reply:
I marked it as handled since MenuBar was also handling it and moving focus to one more than what we wanted, so it was "jumping" one element.

Reply:
You should only mark it as handled for the keys that you handle (left, right, down, enter, space), not for all keys (e.g. 'K' or whatever).